### PR TITLE
Hubot script to accept the room name

### PR DESCRIPTION
**Pull request by thinkeridea**
We expect in messages received or room name, rather than the room id, because the room id cannot access to the page and not easy to understand [#213](https://github.com/RocketChat/hubot-rocketchat/issues/213)

### CODE REVIEW

**Comment by JigSawFr** [comment]
Linked to https://github.com/RocketChat/Rocket.Chat/pull/7102

File: src/rocketchat.coffee [hunk 1]
```diff
@@ -155,11 +155,12 @@ class RocketChatBotAdapter extends Adapter
 						@lastts = curts
 
 						user = @robot.brain.userForId newmsg.u._id, name: newmsg.u.username, alias: newmsg.alias
-						user.room = newmsg.rid
 
-						if newmsg.t is 'uj'
-							@robot.receive new EnterMessage user, null, newmsg._id
-						else
+						@chatdriver.getRommName(newmsg.rid).then((roomName)=>
```
Comment (geekgonecrazy) [inline src/rocketchat.coffee]: Can we fix the spelling on this to `getRoomName` ?

Thanks!

File: src/rocketchat_driver.coffee [hunk 2]
```diff
@@ -39,6 +40,9 @@ class RocketChatDriver
 	getRoomId: (room) =>
 		@tryCache _roomIdCache, 'getRoomIdByNameOrId', room, 'Room ID'
 
+	getRommName: (room) =>
```
Comment (geekgonecrazy) [inline src/rocketchat_driver.coffee]: Same fix here of course :)

File: src/rocketchat_driver.coffee [hunk 3]
```diff
@@ -126,12 +130,12 @@ class RocketChatDriver
...
-			# awkward syntax due to asteroid limitations
-			# - it ain't no minimongo cursor
-			# @logger.info "Change received on ID " + id
+# awkward syntax due to asteroid limitations
+# - it ain't no minimongo cursor
+# @logger.info "Change received on ID " + id
 			changedMsgQuery = @messages.reactiveQuery {"_id": id}
 			if changedMsgQuery.result && changedMsgQuery.result.length > 0
-				# console.log('result:', JSON.stringify(changedMsgQuery.result, null, 2))
+# console.log('result:', JSON.stringify(changedMsgQuery.result, null, 2))
```
Comment (geekgonecrazy) [inline src/rocketchat_driver.coffee]: Can we also get these comments moved back in line like they were?

Comment (thinkeridea) [inline src/rocketchat_driver.coffee]: I'm sorry I use the ide automatically formatted

**Comment by jangmarker** [comment]
Can this be merged? I think v57 of Rocket.chat has the necessary API (https://github.com/RocketChat/Rocket.Chat/releases/tag/0.57.0), doesn't it?

**Pull request merged**